Sources/TuistSupport/UI/UIController.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+import Noora
+
+/// A controller for UI elements on the terminal.
+public struct UIController: Noorable {
```

Comment:
This one is extending by wrapping to:
- Add a new capability (stateless raw output).
- Conditionally output based on an attribute.

I think those two things are candidates to be capabilities of [Noora](https://github.com/tuist/noora):

```swift
let noora = Noora(quiet: true)
noora.raw("\(jsonGraph)")
```

> [!NOTE]
> Nora does internal logging even in the case of `quiet: true`, which is useful for debugging purposes.

Reply:
I agree, it would be nice to have this functionality directly available in Noora. Then the controller becomes obsolete. I added some TODOs in the file,  to not loose sight of the upcoming changes, is this ok ?

Reply:
@alexanderwe should we instead implement first the changes in Noora before going ahead with this PR? Or do you think the `UIController` would be needed even with those changes in Noora?

Reply:
@fortmarek As soon as Noora provides the possibility to be "quiet" and has a "raw" output format I think it is not necessary to have the `UIController` wrapper. 

If we add the `isQuiet` property to Noora I think we also need a way to override it. e.g. for the case where we want to raw output the JSON but still everything else should be quiet.

Reply:
In terms of if we should first add the changes to Noora depends a little bit on when you would like to see this changes land in `main` 😄

Reply:
We're not in a rush. I'd lean to first doing the changes in Noora and then going back to this PR. What do you think, @alexanderwe?

Reply:
Then let's wait until the changes land in Noora, then the complexity of the codebase is not increased and the number of LOC can be reduced here.